core/modules/admin_bar/admin_bar.module

```diff
@@ -244,6 +245,21 @@ function admin_bar_preprocess_page(&$variables) {
     $settings['position_fixed'] = $setting;
   }
 
+  // Send whether this is an admin page to JS for back to site functionality.
+  $settings['current_path_is_admin'] = FALSE;
+  $current_path = current_path();
+  if (path_is_admin($current_path)) {
+    $settings['current_path_is_admin'] = TRUE;
+  }
+  $alias = backdrop_get_path_alias($current_path, $language->langcode);
+  $settings['current_path'] = $alias;
+
+  $components = $config->get('components');
+  if (in_array('admin_bar.back_to_site_link', $components)) {
+    $settings['back_to_site_link'] = TRUE;
+  }
+
+  $settings['frontpage'] = config_get('system.core', 'site_frontpage');
```

Comment:
What is the purpose of this? `frontpage` is not used anywhere...

Reply:
I think this was for when back to site didn’t replace home but I’ll make sure before I remove it.

Reply:
Yes this was not necessary.